docs/docs/deploying-to-qovery.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Deploying to Qovery
+---
+
+## Introduction
+
+[Qovery](https://www.qovery.com) is a fully-managed Cloud platform where you can host your modern application (web, database, broker, storage).
+
+> Trusted by developers from great companies - Google, Red Hat, Elastic and more
+
+Qovery include the following:
+
+- Continuous, automatic builds & deploys from GitHub, Gitlab and Bitbucket.
+- [Custom domains](https://docs.qovery.com/services/network/dns).
+- Automatic TLS certificates through [Let's Encrypt](https://letsencrypt.org).
+- Microservices support.
+- Auto-scaling support.
+- Native HTTP/2 support.
+- [Isolated environment](https://docs.qovery.com/extending-qovery/branches-and-environments) per git branch.
+- Managed databases (PostgreSQL, MySQL, MongoDB, Elasticsearch, Redis, Memcached, Cassandra).
+- Managed brokers (RabbitMQ, Kafka).
+- Managed storage (S3).
+- Custom URL redirects and rewrites.
+
+## Prerequisites
+To get started, you'll need:
+
+- You have an account with GitHub, GitLab or Bitbucket.
+- You have completed the Quick Start or have a Gatsby website you are ready to deploy and share with the world.
+- You have a Qovery account. You can sign up at [start.qovery.com](https://start.qovery.com).
+
+## Deploy your application
+
+You can set up a Gatsby site on Qovery in five quick steps:
+
+**1/** Install the Qovery CLI
+Here is [how to install the Qovery CLI](https://docs.qovery.com/extending-qovery/cli) documentation
+
+**2/** Sign up
+
+```bash
+# Sign up with Github, Gitlab, Bitbucket
+$ qovery auth
+```
+
+**3/** Configure your project and generate a .qovery.yml
+
+```bash
+# generate the .qovery.yml at the root of your project directory
+$ qovery init
+```
+
+**4/** Add this Dockerfile at the root of your repository
```

Comment:
Is it not sufficient to say "Add this Dockerfile at the root of your repository" ?

Reply:
I apologise for being unclear . I was suggesting adding a paragraph that states something like “ you can achieve this by running “touch docker-compose.yml” in your terminal and add the code block below to this file “ .  Since the above steps suggests these actives are carried out in the terminal . 

We try to be explicit with instructions while making guides to reduce assumptions. ( if any assumptions are made about the user’s knowledge of docker, it should be clearly stated at the start of the guild )

Reply:
Yes, pretty clear. Thank you @Ekwuno 👍